app/views/admin/participants/validation_data/full_name.html.erb

```diff
@@ -0,0 +1,14 @@
+<% title = "What is #{possessive_name(@validation_data_form.participant_name)} full name as registered with the TRA?" %>
+<% content_for :title, title %>
+<% content_for :before_content, govuk_back_link( text: "Back", href: admin_participant_path(id: @participant_profile)) %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl">Validation data - <%= @validation_data_form.participant_name %></span>
+    <h1 class="govuk-heading-xl"><%= title %></h1>
```

Comment:
Maybe the same here, is the heading and label superflouous?

Reply:
Yes, agree. Same on all pages.